persistent/list.py

```diff
@@ -55,8 +73,19 @@ def __setslice__(self, i, j, other):
             self._p_changed = 1
 
         def __delslice__(self, i, j):
+            # On Python 2, the slice dunders, like ``__delslice__``,
+            # are documented as getting ``(0, sys.maxsize)`` as the
+            # arguments for list[:]. Prior to 2.7.10, it was
+            # incorrectly documented as ``(0, sys.maxint)``, but that
+            # usually worked because ``sys.maxsize`` and
+            # ``sys.maxint`` are usually the same. But on 64-bit
+            # Windown, where ``sizeof(int) == sizeof(long) == 4``,
+            # they're different (``sys.maxsize`` is bigger). See
+            # https://bugs.python.org/issue23645
+            needs_changed = i == 0 and j == sys.maxsize and bool(self)
```

Comment:
I think this change now masks "normal" list slice deetions (e.g., `del user_list[3:4]`) as being marked changed.